src/background/services/tabEvents.ts

```diff
@@ -87,7 +87,7 @@ export class TabEvents {
      * if loading and no url -> clear all sessions but not the overpaying state
      * if loading and url -> we need to check if state keys include this url.
      */
-    if (changeInfo.status === 'loading' && changeInfo.url) {
+    if (changeInfo.status === 'loading') {
```

Comment:
This reverts and breaks https://github.com/interledger/web-monetization-extension/pull/831, until we've a better solution to distinguish between page reload and iframe navigation on page.

TODO: decide with team which is better breakage. Actual fix (to be done) will require substantial effort.

Reply:
i.e. 
- Don't clear sessions on iframe navigation (cursed)
- Don't clear sessions on page refresh, creating cumulatively more outgoing payments after reach each refresh (exponentially cursed)